raiden/tests/integration/test_stress.py

```diff
@@ -0,0 +1,380 @@
+            server_to,
+            deposit,
+            token_address,
+            identifier_generator,
+        )
+
+
+def stress_send_parallel_transfers(rest_apis, token_address, identifier_generator, deposit):
+    """Send `deposit` transfers in parallel, without changing the initial capacity.
+    """
+    pairs = list(zip(rest_apis, rest_apis[1:] + [rest_apis[0]]))
+
+    # deplete the channels in one direction
+    gevent.wait([
+        gevent.spawn(
+            sequential_transfers,
+            server_from,
+            server_to,
+            deposit,
+            token_address,
+            identifier_generator,
+        )
+        for server_from, server_to in pairs
+    ])
+
+    # deplete the channels in the backwards direction
+    gevent.wait([
+        gevent.spawn(
+            sequential_transfers,
+            server_from,
+            server_to,
+            deposit * 2,
+            token_address,
+            identifier_generator,
+        )
+        for server_to, server_from in pairs
+    ])
+
+    # reset the balances balances by sending the "extra" deposit forward
+    gevent.wait([
+        gevent.spawn(
+            sequential_transfers,
+            server_from,
+            server_to,
+            deposit,
+            token_address,
+            identifier_generator,
+        )
+        for server_from, server_to in pairs
+    ])
+
+
+def stress_send_and_receive_parallel_transfers(
+        rest_apis,
+        token_address,
+        identifier_generator,
+        deposit,
+):
+    """Send transfers of value one in parallel"""
+    pairs = list(zip(rest_apis, rest_apis[1:] + [rest_apis[0]]))
+
+    foward_transfers = [
+        gevent.spawn(
+            sequential_transfers,
+            server_from,
+            server_to,
+            deposit,
+            token_address,
+            identifier_generator,
+        )
+        for server_from, server_to in pairs
+    ]
+
+    backwards_transfers = [
+        gevent.spawn(
+            sequential_transfers,
+            server_from,
+            server_to,
+            deposit,
+            token_address,
+            identifier_generator,
+        )
+        for server_to, server_from in pairs
+    ]
+
+    gevent.wait(foward_transfers + backwards_transfers)
+
+
+def assert_channels(raiden_network, token_network_identifier, deposit):
+    pairs = list(zip(raiden_network, raiden_network[1:] + [raiden_network[0]]))
+
+    for first, second in pairs:
+        wait_assert(
+            assert_synced_channel_state,
+            token_network_identifier,
+            first, deposit, [],
+            second, deposit, [],
+        )
+
+
+@pytest.mark.parametrize('number_of_nodes', [3])
+@pytest.mark.parametrize('number_of_tokens', [1])
+@pytest.mark.parametrize('channels_per_node', [2])
+@pytest.mark.parametrize('deposit', [5])
+@pytest.mark.parametrize('reveal_timeout', [15])
+@pytest.mark.parametrize('settle_timeout', [120])
+def test_stress(
```

Comment:
suggestion: Not for this PR, but it would perhaps be easier to have a setting for "number_of_transfers" made with the test instead of all these.

Then depending on that setting the stress test would calculate what number of nodes and what deposit would be needed to test the provided number of transfers.